tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgentManager.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="ServerlessMiniAgentManager.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal class ServerlessMiniAgentManager
+{
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ServerlessMiniAgentManager));
+
+    internal virtual void Start(string path)
+    {
+        try
+        {
+            Log.Debug("Trying to spawn the Serverless Mini Agent at path: {Path}", path);
+            Process process = new Process();
```

Comment:
You may want to save a reference to this `Process` in a class field to prevent the GC from collecting it (although maybe even handler already takes care of that, but just to be safe?).

Also, do you ever need to stop/kill the process?

Reply:
ooh didn't catch that. I'm keeping a reference to Process in `Tracer.cs` now. 

We shouldn't ever need to kill the mini agent, it'll die when azure functions spins down the function process.